Gems/RecastNavigation/Code/Source/Misc/RecastNavigationMeshComponentController.cpp

```diff
@@ -63,6 +63,12 @@ namespace RecastNavigation
 
     void RecastNavigationMeshComponentController::UpdateNavigationMeshBlockUntilCompleted()
```

Comment:
could this return true if the update was successful and false if its already in progress?

Reply:
That is a good idea. Thanks!

Reply:
Done.